app/assets/stylesheets/components/_file-input.scss

```diff
@@ -67,7 +67,7 @@
     padding: 0;
   }
 
-  .usa-file-input__preview__image {
+  .usa-file-input__preview-image {
```

Comment:
This was previously misnamed, fixed between USWDS 2.8.0 and 2.8.1:

- https://github.com/uswds/uswds/issues/3573
- https://github.com/uswds/uswds/commit/de86e3bf8571070d24c370df43de5ffb1650293b